podpac/core/coordinates/coordinates.py

```diff
@@ -360,7 +360,7 @@ def from_xarray(cls, x, crs=None):
                 # unstacked
                 d[dim] = ArrayCoordinates1d.from_xarray(xcoords[dim])
 
-        coords = cls(list(d.values()), crs=crs)
+        coords = cls(list(d.values()), crs=crs, validate_crs=False)

```

Comment:
I think so.

If the crs comes from the DataArray attrs, then presumably it's already been validated. Can we validate if the user specifies the crs and skip if it comes from the DataArray, or do you need to skip it in both cases for the optimization?

```python3
validate_crs = False
if isinstance(x, (xr.DataArray, xr.Dataset)):
    xcoords = x.coords
    if crs is None:
        crs = x.attrs.get("crs")
        validate_crs = True
```

Or, could we provide `validate_crs` as a keyword argument to `from_xarray`, so you can disable it when necessary (like `from_geotransform` below)?

Ultimately, I don't we ever *have* to validate crs at instantiation. We could turn that off everywhere. If it's going to fail, it will fail when transforming during eval, and we can handle that exception then and there.

Reply:
Yes, I'd rather pass it as a flag and have the user make the decision. By default I'll make it False. Thanks! Great suggestion.

Reply:
I suppose we could also make that a setting so we can globally disable crs validation for optimization reasons...